.github/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+<!--
+
+--- PR Hygiene Checklist ---
+
+1. Make sure your branch is from your fork and has a meaningful name
+2. Update the PR title: `BLUEBUTTON-99999 Add Awesomeness`
+3. Edit the text below - do not leave placeholders in the text.
+3.1. Remove sections that you don't feel apply
+4. Add any other details that will be helpful for the reviewers: details description, screenshots, etc
+5. Request a review from someone/multiple someones
+6. <optional> Review your changes yourself and write up any comments / concerns as if you were reviewing someone else's code.
+-->
+
+### Change Details
+
+<!-- Add detailed discussion of changes here: -->
+
+### Security Implications
+
+<!-- Does the change deal with PII/PHI at all? What should reviewers look for in
+terms of security concerns? -->
+
+- [ ] new software dependencies
+
+<!-- If yes, list the new dependencies and briefly note any relevant security impacts -->
+
+- [ ] security controls or supporting software altered
+
+<!-- If yes, what security controls or supporting software are affected? -->
+
+- [ ] new data stored or transmitted
+
+<!-- If yes, what new data are we storing or transmitting? Is the data considered PII/PHI? -->
+
+- [ ] security checklist is completed for this change
+
+<!-- If yes, provide a link to the security checklist in Confluence here. -->
+
+- [ ] requires more information or team discussion to evaluate security implications
+<!-- Use this to indicate you're unsure how this change may impact system security and would like to solicit the team's feedback. Optionally, provide background information regarding your questions and concerns. -->
+
+### Acceptance Validation
```

Comment:
could be renamed to something like "intention". I imagine this section stating the desired effect of the change set as opposed to the details which are how the developer achieved that effect.

Reply:
It's the same as Acceptance Criteria. How can everyone agree that the PR can be done?